developers/bindings/thing-xml.md

```diff
@@ -58,7 +58,7 @@ In that way, a generic thing type could be listed for users and a corresponding
 
 ### Thing Categories
 
-A description about thing categories as well as an overview about which categories exist can be found in our [categories overview](../../concepts/categories.html).
+Thing categories are used to set an icon for that Thing. The available categories correspond with the available icons of the classic icon set, however categories are written in Java class-naming style, e.g. FrontDoor instead of lowercase frontdoor.
```

Comment:
```suggestion
Categories are used to provide meta information about Things. Thing categories describe how the physical device **looks like**. UIs can use this information e.g. to render icons.
The available categories correspond with the [available icons of the classic iconset]({{base}}/configuration/iconsets/classic/), however categories are written in Java class-naming style, e.g. `FrontDoor` instead of lowercase `frontdoor`.
```